app/views/declaration/supervising_office.scala.html

```diff
@@ -30,11 +30,13 @@
 
         @components.error_summary(form.errors)
 
+        @components.section_header(messages("supplementary.summary.locations.header"))
+
         @components.page_title(Some("supplementary.supervisingCustomsOffice.title"))
 
         @components.input_text(
             field = form("supervisingCustomsOffice"),
-            label = messages("supplementary.supervisingCustomsOffice"),
+            label = "",
```

Comment:
The same question here

Reply:
Yes same thing here. No label, but we have a hint.